mlflow/utils/autologging_utils.py

```diff
@@ -119,6 +126,12 @@ def wrap_patch(destination, name, patch, settings=None):
 
     original = getattr(destination, name)
     wrapped = functools.wraps(original)(patch)
+    try:
+        wrapped.__signature__ = inspect.signature(original)
```

Comment:
Unfortunately, this can fail (as it does when attempting to run it on `tensorflow.estimator.Estimator.exported_savedmodel` due to a 'hiddenattribute' concept in TensorFlow)